.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,5 +1,5 @@
 Fixes #[Add issue number here. Note: This will automatically close the issue. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
 
-Changes: [Add here what changes were made in this issue and if possible provide links.]
+Changes: [Please Mention the files changes. Add here what changes were made in this issue and if possible provide a preview/deploy links.]
```

Comment:
`Please Mention the files changes` This is grammatically incorrect. Please use proper grammar.